--- debian/eebackuppc4/etc/backuppc/config.pl	2020-07-31 13:40:20.044504281 +0200
+++ debian/eebackuppc4/etc/backuppc/config.pl	2020-07-31 13:49:38.997844687 +0200
@@ -41,7 +41,10 @@
 #
 # Host name on which the BackupPC server is running.
 #
-$Conf{ServerHost} = 'XXXXXX';
+$ENV{'PATH'} = '/bin:/usr/bin';
+delete @ENV{'IFS', 'CDPATH', 'ENV', 'BASH_ENV'};
+$Conf{ServerHost} = `hostname`;
+chomp($Conf{ServerHost});
 
 #
 # TCP port number on which the BackupPC server listens for and accepts
@@ -303,7 +306,7 @@
 # Full path to various commands for archiving
 #
 $Conf{SplitPath} = '/usr/bin/split';
-$Conf{ParPath}   = '';
+$Conf{ParPath}   = '/usr/bin/par2' if -x '/usr/bin/par2';
 $Conf{CatPath}   = '/bin/cat';
 $Conf{GzipPath}  = '/bin/gzip';
 $Conf{Bzip2Path} = '/bin/bzip2';
@@ -1276,6 +1279,9 @@
 $Conf{RsyncArgs} = [
     '--super',
     '--recursive',
+    # --protect-args does not works with rrsync. Comment
+    # it if you use rrsync. Drawback, special chars in
+    # share names are not allowed.
     '--protect-args',
     '--numeric-ids',
     '--perms',
@@ -1377,6 +1383,9 @@
 $Conf{RsyncRestoreArgs} = [
     '--recursive',
     '--super',
+    # --protect-args does not works with rrsync. Comment
+    # it if you use rrsync. Drawback, special chars in
+    # share names are not allowed.
     '--protect-args',
     '--numeric-ids',
     '--perms',
@@ -1579,7 +1588,7 @@
 # Full path for ssh. Security caution: normal users should not
 # allowed to write to this file or directory.
 #
-$Conf{SshPath} = '/usr/bin/ssh';
+$Conf{SshPath} = '/usr/bin/ssh' if -x '/usr/bin/ssh';
 
 #
 # Full path for nmblookup. Security caution: normal users should not
@@ -2098,7 +2107,7 @@
 # Full URL of the BackupPC_Admin CGI script, or the configured path
 # for SCGI.  Used for links in email messages.
 #
-$Conf{CgiURL} = 'http://XXXXXX/cgi-bin/BackupPC_Admin';
+$Conf{CgiURL} = 'http://'.$Conf{ServerHost}.'/backuppc/index.cgi';
 
 #
 # Full path to the rrdtool command.  If available, graphs of pool usage
